paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerModule.kt

```diff
@@ -101,9 +101,10 @@ internal object FlowControllerModule {
     @Singleton
     fun providesWalletButtonsContent(
         viewModel: FlowControllerViewModel,
+        @Named(WALLETS_BUTTON_LINK_LAUNCHER) walletsButtonLinkLauncher: LinkPaymentLauncher,
     ): WalletButtonsContent {
         return WalletButtonsContent(
-            interactor = DefaultWalletButtonsInteractor.create(viewModel)
+            interactor = DefaultWalletButtonsInteractor.create(viewModel, walletsButtonLinkLauncher)
```

Comment:
had to inject this dependency rather than retrieving it from `viewModel.flowComponent` because seems like exposes properties directly from the component do not respect the name qualifier